diff --git a/config/config.pl b/config/config.pl
--- a/config/config.pl
+++ b/config/config.pl
@@ -2058,12 +2058,12 @@
 else {
   # similar to the perl installation procedure, append latex2html only
   # if it does not appear in the PREFIX
-  if($newcfg{'PREFIX'} =~ /l2h|latex2html/i) {
-    $newcfg{'SHLIBDIR'} = $newcfg{'PREFIX'};
-  }
-  else {
+#  if($newcfg{'PREFIX'} =~ /l2h|latex2html/i) {
+#    $newcfg{'SHLIBDIR'} = $newcfg{'PREFIX'};
+#  }
+#  else {
     $newcfg{'SHLIBDIR'} = "$newcfg{'PREFIX'}${dd}share${dd}lib${dd}latex2html";
-  }
+#  }
 }
 add_drive_letter($newcfg{'SHLIBDIR'});
 
