--- latex2html-2008/latex2html.pin.bero	2012-03-26 17:15:55.431687858 +0200
+++ latex2html-2008/latex2html.pin	2012-03-26 17:21:24.713706594 +0200
@@ -532,7 +532,7 @@ $CHARSET = $charset || 'iso-8859-1';
 #
 # If possible, use icons of the same type as generated images
 #
-if ($IMAGE_TYPE && defined %{"icons_$IMAGE_TYPE"}) {
+if ($IMAGE_TYPE && %{"icons_$IMAGE_TYPE"}) {
     %icons = %{"icons_$IMAGE_TYPE"};
 }
 
@@ -2112,7 +2112,7 @@ sub convert_iso_latin_chars {
 		"\nCould not find translation function for $default_language.\n\n")
 	    }
 	);
-	if ($USE_UTF ||(!$NO_UTF &&(defined %unicode_table)&&length(%unicode_table)>2)) {
+	if ($USE_UTF ||(!$NO_UTF &&(%unicode_table)&&length(%unicode_table)>2)) {
 	    &convert_to_unicode($_)};
     }
     $_ = join('', @case_processed, $_); undef(@case_processed);
@@ -9117,7 +9117,7 @@ sub real_replace_strange_accents {
     my ($charset) = "${CHARSET}_character_map_inv";
     $charset =~ s/-/_/g;
     # convert upper 8-bit characters
-    if (defined %$charset &&($CHARSET =~ /8859[_\-]1$/)) {
+    if (%$charset &&($CHARSET =~ /8859[_\-]1$/)) {
 	s/([\200-\377])/
 	    $tmp = $$charset{'&#'.ord($1).';'};
 	    &mark_string($tmp) if ($tmp =~ m!\{!);
@@ -10618,19 +10618,17 @@ sub do_rightquotes { (($HTML_VERSION < 5
 sub do_cmd_parbox {
     local($_) = @_;
     local($args, $contents, $dum, $pat);
-    $* = 1;			# Multiline matching ON
     ($dum,$pat) = &get_next_optional_argument; # discard this
     ($dum,$pat) = &get_next_optional_argument; # discard this
     ($dum,$pat) = &get_next_optional_argument; # discard this
     $args .= $pat if ($pat);
     $pat = &missing_braces unless (
-	(s/$next_pair_pr_rx/$pat=$2;''/eo)
-	||(s/$next_pair_rx/$pat=$2;''/eo));
+	(s/$next_pair_pr_rx/$pat=$2;''/eom)
+	||(s/$next_pair_rx/$pat=$2;''/eom));
     $args .= "{".$`.$pat."}";
     $contents = &missing_braces unless (
-	(s/$next_pair_pr_rx/$contents=$2;''/eo)
-	||(s/$next_pair_rx/$contents=$2;''/eo));
-    $* = 0;			# Multiline matching OFF
+	(s/$next_pair_pr_rx/$contents=$2;''/eom)
+	||(s/$next_pair_rx/$contents=$2;''/eom));
     $args .= "{".$`.$contents."}";
     if ($NO_PARBOX_IMAGES) {
 	$contents = join ('', &do_cmd_par(), $contents, '</P>' );
