--- latex2html-2002-2-1/config/config.pl.tv	2003-11-21 11:25:13.000000000 +0100
+++ latex2html-2002-2-1/config/config.pl	2003-11-21 13:32:32.000000000 +0100
@@ -617,7 +617,7 @@
 
 # only perl gets an absolute pathname
 my $abs_path_names = 1;
-$newcfg{'PERL'} = &find_prog($^X); # take the perl internal executable name
+$newcfg{'PERL'} = &find_prog("perl"); # take the perl internal executable name
 &checking('perl version');
 &result($]);
 
