--- latex2html-2002-1/latex2html.pin.orig	Wed May  1 07:15:01 2002
+++ latex2html-2002-1/latex2html.pin	Tue Aug  6 11:26:12 2002
@@ -457,12 +457,12 @@
 }
 if ($opt{test_mode}) {
     $TITLE = 'LaTeX2HTML Test Document';
-    $TEXEXPAND = "$PERL @srcdir@${dd}texexpand@scriptext@";
-    $PSTOIMG   = "$PERL @srcdir@${dd}pstoimg@scriptext@";
-    $ICONSERVER = L2hos->path2URL("@srcdir@${dd}icons");
+    $TEXEXPAND = "@PERLSCRIPTDIR@${dd}texexpand@scriptext@";
+    $PSTOIMG   = "@PERLSCRIPTDIR@${dd}pstoimg@scriptext@";
+    $ICONSERVER = L2hos->path2URL("@LATEX2HTMLDIR@${dd}icons");
     $TEST_MODE  = 1;
-    $RGBCOLORFILE = "@srcdir@${dd}styles${dd}rgb.txt";
-    $CRAYOLAFILE = "@srcdir@${dd}styles${dd}crayola.txt";
+    $RGBCOLORFILE = "@LATEX2HTMLDIR@${dd}styles${dd}rgb.txt";
+    $CRAYOLAFILE = "@LATEX2HTMLDIR@${dd}styles${dd}crayola.txt";
 }
 if($DEBUG) {
     # make the OS-dependent functions more chatty, too
