--- latex2html-2002/config/config.pl.orig	2002-04-10 07:15:01.000000000 +0200
+++ latex2html-2002/config/config.pl	2002-04-12 22:12:12.000000000 +0200
@@ -1143,7 +1143,7 @@
               push(@right_paths,L2hos->path2os($testpath));
               $gs_lib = 1;
             }
-            $testpath .= "${dd}fonts" unless($testpath =~ /\Q$dd\Efonts$/i);
+            $testpath .= "${dd}" unless($testpath =~ /\Q$dd\Efonts$/i);
             if(!$gs_fonts && -d $testpath) {
               push(@right_paths,L2hos->path2os($testpath));
               $gs_fonts = 1;
